client/utils.go

```diff
@@ -13,6 +14,23 @@ import (
 
 var headerRegexp = lazyregexp.New(`\ADocker/.+\s\((.+)\)\z`)
 
+type emptyIDError string
+
+func (e emptyIDError) InvalidParameter() {}
+
+func (e emptyIDError) Error() string {
+	return "invalid " + string(e) + " name or ID: value is empty"
+}
```

Comment:
Could use feedback on this; currently this produces;

```
invalid <object type> name or ID: value is empty
```

for example;

```
invalid container name or ID: value is empty
```

I was trying to find a generic term for this, and now all objects may have an "ID" (volumes, e.g.); perhaps something like "reference" or "identifier" would work?

```
invalid container reference: value is empty
```

```
invalid container identifier: value is empty
```